src/modrover/learner.py

```diff
@@ -242,6 +236,3 @@ def score(self, test_set: DataFrame, model: Optional[RegmodModel] = None) -> flo
         model.data.detach_df()
 
         return performance
-
-    def __repr__(self):
```

Comment:
Although the learner ID serves no purpose in the learner instance's operation, it might be useful for printing/indentification/etc.